src/UseCase/BannerProvideUseCase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\UseCase;
+
+use Symfony\Component\HttpFoundation\Response;
```

Comment:
Wrong response type. The use case should be isluated from the framework. That means it should have its own response value object and the controller transforms that value object into HTTP headers, cookies and response body.